data/sql/grid_alternatives.sql

```diff
@@ -116,6 +116,10 @@ VALUES
 ('EGM08_REDNAP.txt','es_ign_egm08-rednap.tif',NULL,'GTiff','geoid_like',0,NULL,'https://cdn.proj.org/es_ign_egm08-rednap.tif',1,1,NULL),
 ('EGM08_REDNAP_Canarias.txt','es_ign_egm08-rednap-canarias.tif',NULL,'GTiff','geoid_like',0,NULL,'https://cdn.proj.org/es_ign_egm08-rednap-canarias.tif',1,1,NULL),
 
+-- eur_nkg - Nordic Geodetic Commission
+('NOT-YET-IN-GRID-TRANSFORMATION-eur_nkg_nkgrf03vel_realigned.tif','eur_nkg_nkgrf03vel_realigned.tif',NULL,'GTiff','velocity_grid',0,NULL,'https://cdn.proj.org/eur_nkg_nkgrf03vel_realigned.tif',1,1,NULL),
```

Comment:
The NOT-YET-IN-GRID-TRANSFORMATION- prefix looks inappropriate.

Reply:
I'm agreed. But without the NOT-YET-IN-GRID-TRANSFORMATION-prefix proj.db build fails (raised by grid_alternatives_insert_trigger). grid_alternatives_insert_trigger should may be triggered in the end of the proj.db build?

Reply:
ah, I believe nkg.sql should be moved after ignf.sql and before grid_alternatives.sql in data/Makefile.am  and data/sql_filelist.cmake

Reply:
Okey. I will test if that works.